frontend/util.lua

```diff
@@ -758,6 +758,11 @@ function util.pathExists(path)
     return lfs.attributes(path, "mode") ~= nil
 end
 
+--- Checks if the given directory exists.
+function util.directoryExists(path)
+  if lfs.attributes(path, "mode") == "directory" then return true end
```

Comment:
See pathExists just above, you can lose much of the syntax fluff and just `return stuff == thing`.

Reply:
Thanks! I learnt a lot about concise coding :D